zhaquirks/smartwings/wm25lz.py

```diff
@@ -0,0 +1,105 @@
+"""Device handler for Smartwings blinds."""
+from zigpy.profiles import zha
+from zigpy.quirks import CustomCluster, CustomDevice
+from zigpy.zcl.clusters.closures import WindowCovering
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+)
+
+from zhaquirks import DoublingPowerConfigurationCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+
+
+class InvertedWindowCoveringCluster(CustomCluster, WindowCovering):
+    """WindowCovering cluster implementation.
+
+    This implementation inverts the reported covering percent for non standard
+    devices that don't follow the reporting spec.
```

Comment:
I guess this comment should be changed to only mention the inverted commands, as the reporting seems to be fine(?)